docs/architecture/blazor-for-web-forms-developers/components.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: Building reusable UI components with ASP.NET Web Forms and Blazor
-description: Building reusable UI components with ASP.NET Web Forms and Blazor.
+title: Building reusable UI components with Blazor
+description: Learn how to build reusable UI components with Blazor and how they compare to ASP.NET Web Form controls.
 author: danroth27
 ms.author: daroth
 ms.date: 09/11/2019
 ---
 
-# Building reusable UI components with ASP.NET Web Forms and Blazor
+# Building reusable UI components with Blazor
```

Comment:
@mairaw Is our eBook guidance any different than docs? Docs discourage the use of gerunds in headings.

Reply:
Fixed

Reply:
We should follow the same guidance to the best of our abilities.